SpatialGDK/Source/SpatialGDKTests/Examples/SpatialGDKExampleTest.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "Misc/ScopeTryLock.h"
+#include "HAL/PlatformFilemanager.h"
+
+#define EXAMPLE_SIMPLE_TEST(TestName) \
+	TEST(SpatialGDKExamples, SimpleExamples, TestName)
+
+#define EXAMPLE_COMPLEX_TEST(TestName) \
+	COMPLEX_TEST(SpatialGDKExamples, ComplexExamples, TestName)
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKExamples, Log, All);
+DEFINE_LOG_CATEGORY(LogSpatialGDKExamples);
+
+// 1. Latent command example
+namespace
+{
+	const double MAX_WAIT_TIME_FOR_SLOW_COMPUTATION = 2.0;
+	const double MIN_WAIT_TIME_FOR_SLOW_COMPUTATION = 1.0;
+	const double COMPUTATION_DURATION = 1.0;
+
+	struct
+	{
+		FCriticalSection Mutex;
+		int Value;
+	} ComputationResult;
+}
+
+DEFINE_LATENT_COMMAND(StartSlowComputation)
```

Comment:
Slow computations seems like the kind of thing we want to discourage in unit tests.

Reply:
That's fair, I'll rename the function.

Reply:
Renamed to `StartBackgroundThreadComputation`